RELEASE_NOTES.md

```diff
@@ -1,17 +1,52 @@
 _This file holds "in progress" release notes for the current release under development and is intended for consumption by the Chef Documentation team. Please see <https://docs.chef.io/release_notes.html> for the official Chef release notes._
 
-# Unreleased
+# Chef Client Release Notes 13.12
 
-## Ohai Release Nots 13.12
+## Smaller Package and Install Size
+
+Both Chef packages and on disk installations have been greatly reduced in size by trimming unnecessary installation files. This has reduced our package size on macOS/Linux by ~50% and Windows by ~12%. With this change Chef 13 is now smaller than a legacy Chef 10 package.
+
+## macOS Mojave (10.14)
+
+Chef is now tested against macOS Mojave, and packages are now available at downloads.chef.io.
+
+## SUSE Linux Enterprise Server 15
+
+- Ohai now properly detects SLES 15
+- The Chef package will no longer remove symlinks to chef-client and ohai when upgrading on SLES 15
+
+## Updated Chef-Vault
+
+Multiple bugfixes in Chef Vault have been resolved by updating chef-vault to 3.4.2
+
+## Faster Windows Installations
+
+Windows installation speeds have been improved by using skipping unnecessary steps when Windows Installer 5+ is available.
```

Comment:
Improved Windows installation speed by skipping unnecessary steps when Windows Installer 5+ is available.